src/block/icon-label/index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'stackable_render_block_icon_label' ) ) {
+	/**
+	 * Adds a special class if block uses the old icon gap attribute
+	 */
+	function stackable_render_block_icon_label( $block_content, $block ) {
+		$attributes = $block[ 'attrs' ];
+
+		// check if the block uses the old icon gap attribute
+		$use_old_styles = ( isset( $attributes[ 'iconGap' ] ) || isset( $block[ 'innerBlocks' ][0]['attrs']['iconSize'] ) ) && ! isset( $attributes[ 'iconGap2' ] );
```

Comment:
Check first for the presence of the elements in `$block['innerBlocks']` to prevent fatal errors.